apps/mull-ui/src/app/components/settings-button/settings-button.tsx

```diff
@@ -5,15 +5,16 @@ import './settings-button.scss';
 
 export interface setttingsButtonProps {
   icon?: IconProp;
-  settingName?: string;
+  text?: string;
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
 }
 
-export const SettingsButton = ({ icon, settingName }: setttingsButtonProps) => {
+export const SettingsButton = ({ icon, text, onClick }: setttingsButtonProps) => {
```

Comment:
This is now a generic profile button, right?

Reply:
Honestly yeah the name isn't perfect, but ProfileButton wouldn't fit either, since its also used in the settings page